cssGrid/css-grid_image-gallery/css/styles.css

```diff
@@ -24,7 +24,9 @@ body {
 .teaser-grid {
   min-width: 40rem;
   min-height: 50vh;
-
+  display: grid;
+  grid-template-columns: repeat(2, 400px);
+  /*magic numbers to make it look pretty*/
```

Comment:
The order is correct but I think your column widths are incorrect? They seem too narrow. The one from you is on the left of the screenshot.

![Screenshot 2023-08-16 at 20 57 14](https://github.com/AntKarpo/challenges/assets/33845176/685ae5de-9704-4c2a-8567-198c1173982a)